README.md

```diff
@@ -87,14 +87,11 @@ Move the report CSVs into this repo, run a script to update Pulse's data, and ma
 ```yaml
 data:
   domains: 2015-03-15
-  dap: 2015-05-29
   scan: 2015-06-07
 ```
 
 `domains`: The date the `.gov` domain list was *generated* by the `.gov` registry.
 
-`dap`: The date the DAP participation list was *generated* by the Digital Analytics Program.
```

Comment:
This isn't new -- we forgot to remove the `dap` field from the README after we automated the generation and download of DAP data and removed it from `meta.yml`.